src/Dapr.Actors.AspNetCore/DaprActorSetupFilter.cs

```diff
@@ -17,17 +16,22 @@ public Action<IApplicationBuilder> Configure(Action<IApplicationBuilder> next)
             // Adds routes for Actors interaction.
             return app =>
             {
-                var actorRouteBuilder = new RouteBuilder(app);
-                actorRouteBuilder.AddDaprConfigRoute();
-                actorRouteBuilder.AddDaprHealthzRoute();
-                actorRouteBuilder.AddActorActivationRoute();
-                actorRouteBuilder.AddActorDeactivationRoute();
-                actorRouteBuilder.AddActorMethodRoute();
-                actorRouteBuilder.AddReminderRoute();
-                actorRouteBuilder.AddTimerRoute();
+                app.UseRouting();
 
-                app.UseRouter(actorRouteBuilder.Build());
+                // This allows the middlewares to be configured correctly from Startup class.
                 next(app);
+
+                // Configure endpoints for Actors.
+                app.UseEndpoints(endpoints =>
```

Comment:
this and L19 is the thing I tried to warn you about 😆  - you can't put `next(app)` in between `UseRouting` and `UseEndpoints`. They have to be a pair.

Reply:
Fixed.